libcxx/test/std/utilities/smartptr/unique.ptr/unique.ptr.create/make_unique_for_overwrite.default_init.pass.cpp

```diff
@@ -27,6 +27,9 @@ constexpr char pattern = 0xDE;
 
 void* operator new(std::size_t count) {
   void* ptr = std::malloc(count);
+  if (!ptr) {
+    std::abort(); // placate MSVC's unchecked malloc warning
```

Comment:
assert?

Reply:
Same reason, `assert(ptr);` doesn't silence the warning.

Reply:
Maybe mention that in the comment, to avoid somebody to beautify this code in the future.

Reply:
And braces, so they all look the same and clang-format is happy.